### ISSUE TITLE
Missing the .empty() method of jQuery =)

### ISSUE THREAD

**Issue by exside** [DanZiti/GrindstoneJS#6]
Would be nice to have something like that, pure js:

```
while(el.firstChild) el.removeChild(el.firstChild); // whit el = an object
```

or similar with grindstone
```
$(el).html('')
```

but this has the issue of possible memory leaks with data and added properties from nodes that are replaced by an empty string

**Comment by dzervoudakes** [comment]
Hey-

Not sure when I'll be able to get to this.  Think you'd want to give it a shot and submit a merge request?

**Comment by dzervoudakes** [comment]
This has been implemented - will be released with v3.1.6 soon, hopefully by next month or earlier.
